internal/commands/relationship.go

```diff
@@ -491,7 +495,7 @@ func writeRelationshipCmdFunc(operation v1.RelationshipUpdate_Operation, input *
 			}
 
 			if operation != v1.RelationshipUpdate_OPERATION_DELETE {
-				if err := handleCaveatFlag(cmd, rel); err != nil {
+				if err := handleCaveatAndExpirationFlags(cmd, rel); err != nil {
 					return err
 				}
```

Comment:
I'd prefer to handle the different flags inline here - remove `handleCaveatAndExpirationFlags` and have a separate `if` statement here for `handleExpiration`.